database/localdb/localdbtest/test.proto

```diff
@@ -0,0 +1,7 @@
+syntax = "proto3";
+
+package localdbtest;
+
+message TestItem {
+  string value = 1;
+}
```

Comment:
Is this proto only for tests within the localdb package? Maybe it should be in a localdb/internal/localdbtest package instead?